src/ServiceInsight/Framework/Settings/ApplicationConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace ServiceInsight.Framework.Settings
+{
+    using System;
+    using System.Configuration;
+
+    public static class ApplicationConfiguration
+    {
+        public static bool SkipCertificateValidation { get; private set; }
+
+        public static void Initialize()
+        {
+            SkipCertificateValidation = GetSkipCertificateValidation();
+        }
+
+        private static bool GetSkipCertificateValidation()
+        {
+            try
+            {
+                return Convert.ToBoolean(ConfigurationManager.AppSettings["SkipCertificateValidation"]);
```

Comment:
what about

```CSharp
return Convert.ToBoolean(ConfigurationManager.AppSettings["SkipCertificateValidation"] ?? "false");
```

to cover for a default "false" value? That way we can change the exception to only be a valid format check instead of requiring the value to exist in the config file?

Reply:
We're shipping the app.config with the default value. Why have two?

Reply:
Just feels weird to not have a default. I have no strong preference one way or the other. Happy with the changes as is.

Reply:
But there _is_ a default, it resides in the config file. I think having a fallback value, would make it confusing for the user as there's no indication we're using that value.

Reply:
Exception is still

`throw new ConfigurationErrorsException("Application setting 'SkipCertificateValidation' is either missing or cannot be converted to type Boolean.", ex);`

It will never throw if it's missing as per Adam's investigation